packages/@webex/plugin-presence/src/index.ts

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import '@webex/internal-plugin-device';
+
+import {registerPlugin} from '@webex/webex-core';
+import {has} from 'lodash';
+
+import Presence from './presence';
+import config from './config';
+
+export interface IResponse {
+  body: {
+    eventType: string;
+    subject: string;
+    status?: string;
+  };
+}
+
+export interface IPredicate {
+  name: string;
+  direction: string;
+  test(ctx: any, response: IResponse): Promise<boolean>;
+  extract?(response: IResponse): Promise<IResponse>;
+}
+
+export interface ITransform {
+  name: string;
+  direction: string;
+  fn(ctx: any, response: IResponse): void;
+}
```

Comment:
Can we put add it along with other interfaces in the interface file and just export from there?

Reply:
Yes. Done